src/main/scala/org/learningconcurrency/ch1/Exercises.scala

```diff
@@ -0,0 +1,52 @@
+// Each of the exercises below provides a function skeleton that you should replace.
+// Below each function is a set of assertions. Your goal is to make them all evaluate to true.
+
+object Exercises extends App {
```

Comment:
I honestly have no idea what that will do since the .sc file is an IntelliJ scratch file. If someone can test what replace the .sc contents with this file's contents would do, I would appreciate that and can then answer your question as to how best to de-duplicate.